tests/testhttp.nim

```diff
@@ -82,77 +21,6 @@ proc continuousTest(address: string, port: Port): Future[int] {.async.} =
       result += 1
     await client.close()
 
-proc customMessage(address: TransportAddress,
-                   data: string,
-                   expect: int): Future[bool] {.async.} =
-  var buffer = newSeq[byte](BufferSize)
-  var header: HttpResponseHeader
-  var transp = await connect(address)
-  defer: transp.close()
-
-  let wres = await transp.write(data)
-  doAssert(wres == len(data))
-  let rres = await transp.readUntil(addr buffer[0], BufferSize, HeadersMark)
-  doAssert(rres > 0)
-  buffer.setLen(rres)
-  header = parseResponse(buffer)
-  doAssert(header.success())
-  return header.code == expect
-
-proc headerTest(address: string, port: Port): Future[bool] {.async.} =
-  var a = resolveTAddress(address, port)
-  var header = "GET / HTTP/1.1\r\n"
-  var i = 0
-  while len(header) <= BigHeaderSize:
-    header.add("Field" & $i & ": " & $i & "\r\n")
-    inc(i)
-  header.add("Content-Length: 71\r\n")
-  header.add("Content-Type: application/json\r\n")
-  header.add("Connection: close\r\n\r\n")
-  header.add("{\"jsonrpc\":\"2.0\",\"method\":\"myProc\",\"params\":[\"abc\", [1, 2, 3]],\"id\":67}")
-  return await customMessage(a[0], header, 413)
-
-proc bodyTest(address: string, port: Port): Future[bool] {.async.} =
-  var body = repeat('B', BigBodySize)
-  var a = resolveTAddress(address, port)
-  var header = "GET / HTTP/1.1\r\n"
-  header.add("Content-Length: " & $len(body) & "\r\n")
-  header.add("Content-Type: application/json\r\n")
-  header.add("Connection: close\r\n\r\n")
-  header.add(body)
-  return await customMessage(a[0], header, 413)
-
-proc disconTest(address: string, port: Port,
-                number: int, expect: int): Future[bool] {.async.} =
-  var a = resolveTAddress(address, port)
-  var buffer = newSeq[byte](BufferSize)
-  var header: HttpResponseHeader
-  var transp = await connect(a[0])
-  defer: transp.close()
-
-  let data = Requests[number]
-  let wres = await transp.write(data)
-  doAssert(wres == len(data))
-  let rres = await transp.readUntil(addr buffer[0], BufferSize, HeadersMark)
-  doAssert(rres > 0)
-  buffer.setLen(rres)
-  header = parseResponse(buffer)
-  doAssert(header.success())
-  if header.code != expect:
-    return false
-
-  let length = header.contentLength()
-  doAssert(length > 0)
-  buffer.setLen(length)
-  await transp.readExactly(addr buffer[0], len(buffer))
-  let left = await transp.read()
-  return len(left) == 0 and transp.atEof()
-
-proc simpleTest(address: string, port: Port,
-                number: int, expect: int): Future[bool] {.async.} =
-  var a = resolveTAddress(address, port)
-  result = await customMessage(a[0], Requests[number], expect)
-
```

Comment:
we're not testing HTTP Server + Client anymore, we just assume that's done in `nim-chronos`